### PR TITLE
Don't allude to mutating the Scala License

### PR DESCRIPTION
**Pull request by dwijnand**
Inspired by the comments from fommil in https://gitter.im/scala/contributors?at=5afc0c2e41f54f22e233305e

WDYT?

### CODE REVIEW

File: _posts/2018-05-15-license-change.md [hunk 1]
```diff
@@ -5,27 +5,27 @@ title: "Intent to adopt the Apache License 2.0 for Scala"
...
 You have until 10 August 2018 to raise any objections (see below for contact info).
 
 # Is any of this legal advice?
 No.
 
-# Why change the license?
+# Why the change?
 A number of contributors have expressed reservations about the asymmetry between our CLA (Apache v2) and our license (BSD-3).
-This change aligns our license with our CLA.
+This change fix this asymmetry.
```
Comment (heathermiller) [inline _posts/2018-05-15-license-change.md]: fixes*

File: _posts/2018-05-15-license-change.md [hunk 2]
```diff
@@ -5,27 +5,27 @@ title: "Intent to adopt the Apache License 2.0 for Scala"
...
 
-# How is the license different?
+# How is are the licenses different?
```
Comment (heathermiller) [inline _posts/2018-05-15-license-change.md]: How are* the licenses...

**Comment by SethTisue** [comment]
cc @fommil

File: _posts/2018-05-15-license-change.md [hunk 3]
```diff
@@ -5,27 +5,27 @@ title: "Intent to adopt the Apache License 2.0 for Scala"
 by: Jorge Vicente Cantero and Adriaan Moors
 ---
 
-Good news everyone! By popular demand, we intend to change the Scala license
+Good news everyone! By popular demand, we intend to change the license that Scala is distributed under
 from the [3-clause BSD License](https://github.com/scala/scala/blob/2.13.x/LICENSE) to the [Apache License v2.0](https://www.apache.org/licenses/LICENSE-2.0.html).
 You have until 10 August 2018 to raise any objections (see below for contact info).
 
 # Is any of this legal advice?
```
Comment (fommil) [inline _posts/2018-05-15-license-change.md]: I'd move this to a footnote, it's kind of weird and formal as the first point, and I'd say redundant. It's important for lawyers to include these disclaimers in their notes to their clients, or a doctor to a patient, if they are just laying out the options, but I don't think there is any ambiguity here.

**Comment by dwijnand** [comment]
Thanks @heathermiller! (how embarrassing)

**Comment by dwijnand** [comment]
Incorporated Sam's advice on the "legal advice" bullet point, so please re-review given that.

**Comment by heathermiller** [comment]
👍 LGTM! Thank you!

**Pull request merged**